sass/px-dropdown.scss

```diff
@@ -98,14 +90,22 @@ $inuit-enable-btn--bare--primary: true;
   }
   &[disabled] {
     color: var(--px-dropdown-text-color--disabled, $gray4);
+    pointer-events: none;
```

Comment:
`pointer-events: none` means that `cursor: not-allowed` won't work. It's better to just remove the hover state ("focused" in this case, I think) for disabled items.